y-octo-node/tests/doc.spec.mts

```diff
@@ -19,4 +19,70 @@ test("y-octo doc", { concurrency: false }, async (t) => {
   await t.test("doc id should be set", () => {
     equal(doc?.clientId, client_id);
   });
+
+  await t.test("array should be created", () => {
+    let arr = doc?.getOrCreateArray("arr");
```

Comment:
should we just throw if the binding failed to create types? the nullish assertions in everywhere look kind of verbose